README.md

```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on 
+x86_64 and AArch64 (ARM64) which doesn't require such (re-)compilations.  
```

Comment:
Please change "AArch64 (ARM64)" to "AArch64(ARM64)".

Reply:
> Please change "AArch64 (ARM64)" to "AArch64(ARM64)".

Okay!